diff --git a/dist/code-12s-Dff2eEMQ.js b/dist/code-12s-Dff2eEMQ.js
index b945fc9401a1dcdf8f4b00bcc3bbfb76f5b07049..8a9d771b819e956bdc883fc195bf5200d7eb279e 100644
--- a/dist/code-12s-Dff2eEMQ.js
+++ b/dist/code-12s-Dff2eEMQ.js
@@ -131,6 +131,8 @@ const getPresetByExt = (ext)=>{
             return presets.python;
         case 'rs':
             return presets.rust;
+        case 'json':
+            return presets.json;
         default:
             return undefined;
     }
